Description: Ignore the runtime-testsuite/annotations and runtime-testsuite/processors modules
 (the --ignore option in debian/antlr4.poms doesn't work due to #839738)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -87,8 +87,6 @@
 		<module>tool</module>
 		<module>antlr4-maven-plugin</module>
 		<module>tool-testsuite</module>
-		<module>runtime-testsuite/annotations</module>
-		<module>runtime-testsuite/processors</module>
 		<module>runtime-testsuite</module>
 	</modules>
 
